composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "docler-labs/codeception-slim-module",
+    "description": "Codeception Module for Slim framework.",
+    "keywords": [
+        "codeception",
+        "module",
+        "slim"
+    ],
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": "^7.2",
+        "codeception/codeception": "^4.0",
+        "codeception/lib-innerbrowser": "^1.0",
+        "slim/slim": "^3.1"
```

Comment:
Is your library compatible with 4.x versions of Slim? Would be nice to support latest versions and make your library forward-compatible, or at least prepare separate branches for v1 with Slim 3.x and v2 (aka master) for Slim 4.x to avoid later dependency hell.

Reply:
v4 has been completely rewritten and contains many breaking change. Since v3 is still a popular version, the plan is to support v3 first, and create a new major version to support v4. At the end we will have two branch 1.x and 2.x.